cfn-resources/x509-authentication-database-user/mongodb-atlas-x509authenticationdatabaseuser.json

```diff
@@ -1,41 +1,6 @@
 {
     "additionalProperties": false,
     "definitions": {
-        "Certificate": {
-            "type": "object",
-            "properties": {
-                "UserName": {
-                    "type": "string",
-                    "description": " Username of the database user to create a certificate for."
-                },
-                "CreatedAt": {
-                    "type": "string",
-                    "description": "Timestamp in ISO 8601 date and time format in UTC when Atlas created this X.509 certificate."
```

Comment:
Why do we need to remove Certificate?

Reply:
the only place the Certificate was being used was in the "results" since ive decided to remove the results, it doesnt make sense to have this declaration anymore